templates/Alert.json

```diff
@@ -49,6 +49,15 @@
     },
     "appInsightsQuery": {
       "type": "string"
+    },
+    "commonTags": {
+      "type": "string"
+    },
+    "additionalTags": {
+      "type": "object",
+      "defaultValue": {
+        "[concat('hidden-link:', resourceId('microsoft.insights/components', parameters('appInsightsName')))]": "Resource"
```

Comment:
I think this should be in the variables section? is there a reason you made it a paremeter to the template?

Reply:
Migrated to variable